src/common/tools.ts

```diff
@@ -400,17 +400,17 @@ const ONE_MB = ONE_KB * 1024
 const ONE_GB = ONE_MB * 1024
 
 export const transMemorySizeNumToStr = (byte: number, toFixed?: number): string => {
-  const getNumPart = (num: number) => (toFixed ? num.toFixed(toFixed) : num)
+  const getNumPart = (num: number) => (toFixed ? round(num, toFixed) : num)
   if (byte < ONE_KB) {
-    return getNumPart(byte) + 'Byte'
+    return getNumPart(byte) + ' bytes'
   }
   if (byte < ONE_MB) {
-    return getNumPart(byte / ONE_KB) + 'KB'
+    return getNumPart(byte / ONE_KB) + ' KB'
   }
   if (byte < ONE_GB) {
-    return getNumPart(byte / ONE_MB) + 'MB'
+    return getNumPart(byte / ONE_MB) + ' MB'
   }
-  return getNumPart(byte / ONE_GB) + 'GB'
+  return getNumPart(byte / ONE_GB) + ' GB'
 }
```

Comment:
I think it's good to add to shared-ui https://github.com/emqx/shared-ui/tree/main/packages/utils